src/web_connection_base.cpp

```diff
@@ -113,7 +113,11 @@ namespace libtorrent
 
 	void web_connection_base::start()
 	{
-		set_upload_only(true);
```

Comment:
but I suspect the invariant check either shouldn't be called there, or this invariant shouldn't be checked